src/Facebook/FacebookRequestException.php

```diff
@@ -80,7 +80,7 @@ public static function create($raw, $data, $statusCode)
     if (!isset($data['error']['code']) && isset($data['code'])) {
       $data = array('error' => $data);
     }
-    $code = (isset($data['error']['code']) ? $data['error']['code'] : null);
+    $code = (isset($data['error']['code']) ? (int)$data['error']['code'] : null);
```

Comment:
Space between `(int)` and `$data` to be consistent with the rest of the code in the SDK.